src/Utils/InstrumentSchema.php

```diff
@@ -105,7 +97,7 @@ protected static function wrap_fields( array $fields, string $type_name ) {
 			 * @throws Exception
 			 * @since 0.0.1
 			 */
-			$field->resolveFn = function ( $source, array $args, AppContext $context, ResolveInfo $info ) use ( $field_resolver, $type_name, $field_key, $field ) {
+			$field->resolveFn = static function ( $source, array $args, AppContext $context, ResolveInfo $info ) use ( $field_resolver, $type_name, $field_key, $field ) {
```

Comment:
What does the `static` keyword do on this closure? Seems redundant inside a static method.

Reply:
There seems to be some performance impact with explicitly declaring static on closures: https://stackoverflow.com/questions/19899468/php-closures-why-the-static-in-the-anonymous-function-declaration-when-bindin#answer-64481894

That said, my real reasoning for the change was that one of the IDE rules I have setup was yelling at me to declare this static. 🤣

Reply:
Nice! Tks for the link.